default/stringtables/en.txt

```diff
@@ -1797,6 +1797,9 @@ Server option to choose pre-opened file descriptor to accept connections. Works
 OPTIONS_DB_PYTHON_ASYNCIO_INTERVAL
 Interval in seconds to call python asyncio event loop or -1 to disable it. By default asyncio calls disabled.
 
+OPTIONS_DB_AI_EXECUTABLE
+Path to the alternative AI executable. By default use freeorionca{suffix} from the same folder as the server executable.
```

Comment:
"alternative" isn't needed I suppose? This just sets the  path...

Reply:
What does {suffix} mean?

Reply:
That on Windows it's .exe and on Linux and MacOS is empty, not sure how to rephrase it better.

Reply:
I guess something like:

'freeorion' ('freeorion.exe' on Windows)